guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -34,9 +34,11 @@ There are three methods of using iPXE with {ProjectName}:
 . Using PXELinux through TFTP to chainload iPXE directly on bare metal hosts.
 . Using PXELinux through UNDI, which uses HTTP to transfer the kernel and the initial RAM disk on bare-metal hosts.
 
+ifeval::["{build}" == "satellite"]
```

Comment:
Doesn't this also apply to Foreman deployed on RHEL? And since CentOS is just a rebuild without modification, doesn't it also apply to CentOS?

Reply:
Yes but this statement was added so we can refer to it when customer complaints about missing HTTPS, that was the main purpose. In the upstream context, I think it's fine to let the user to find out.

Reply:
I think it still makes sense to warn ahead of time. From experience, I can tell it's quite frustrating to try something and having to figure that out yourself.

Reply:
Sure, let me remove the supported/unsupported statements then.

Reply:
Done